prow/plugins/cat/cat_test.go

```diff
@@ -354,17 +308,9 @@ func TestCats(t *testing.T) {
 			state:         "open",
 			action:        github.GenericCommentActionCreated,
 			body:          "/meow clothes",
-			shouldComment: true,
+			shouldComment: false,
```

Comment:
Given that this is a regression in ux a better response here would be to treat this as equivalent to `/meow` and leave a comment teaching the person that categories are no longer supported. 

Then after a few months switch to this behavior.

Reply:
cat now **chomps** if you request a category